src/utils/storage.ts

```diff
@@ -0,0 +1,51 @@
+interface Storage {
```

Comment:
Maybe use `class` here for consistency?

Reply:
I'll leave at as `interface` since that works, `class` gives me errors and the effort-reward ratio is not high enough to look into it 😅

Reply:
Ah, okay. This might be one of the rare cases Interface is actually needed. Let's leave it then 👍